system/jlib/jstats.cpp

```diff
@@ -853,6 +853,7 @@ static const StatisticMeta statsMetaData[StMax] = {
     { NUMSTAT(AttribsFromCache) },
     { NUMSTAT(SmartJoinDegradedToLocal) },
     { NUMSTAT(SmartJoinSlavesDegradedToStd) },
+    { NUMSTAT(SimplifiedMoreComplex) },
```

Comment:
It would be better to have a consistent name with the others E.g. AttrsSimplifiedTooComplex or something similar.